ros2topic/ros2topic/verb/pub.py

```diff
@@ -44,26 +44,32 @@
 MsgType = TypeVar('MsgType')
 
 
-def set_message_fields_expanded(node: Node, msg: Any, values: Dict[str, str]) -> List[Any]:
+def set_message_fields_expanded(
+        msg: Any, values: Dict[str, str], expand_header_auto: bool = False,
+        expand_time_now: bool = False) -> List[Any]:
     """
     Set the fields of a ROS message.
 
-    This function is copied from :py:func:`rosidl_runtime_py.set_message_fields`, but with a
-    'Node' parameter, required for obtaining the current time.
     :param msg: The ROS message to populate.
     :param values: The values to set in the ROS message. The keys of the dictionary represent
         fields of the message.
-        If 'auto' is passed as a value to a 'std_msgs.msg.Header' field, an empty Header will be
-        instantiated with its 'stamp' field set to the current time.
-        If 'now' is passed as a value to a 'builtin_interfaces.msg.Time' field, its value will be
-        expanded to the current time.
+    :param expand_header_auto: If enabled and 'auto' is passed as a value to a
+        'std_msgs.msg.Header' field, an empty Header will be instantiated and a setter function
+        will be returned so that its 'stamp' field can be set to the current time.
+    :param expand_time_now: If enabled and 'now' is passed as a value to a
+        'builtin_interfaces.msg.Time' field, a setter function will be returned so that
+        its value can be set to the current time.
```

Comment:
this doesn't seem to be accurate for the function defined here (though accurate for the `rosidl_runtime_py` PR).

Reply:
I'm afraid I'm not following, both docstrings are the same. Can you elaborate what should be updated here? Thanks.

Reply:
Sorry, I think I got confused when reading the diff.
The docs are fine